pkg/cluster/nodes/node.go

```diff
@@ -116,12 +112,6 @@ func (n *Node) Name() string {
 	return n.name
 }
 
-// SignalStart sends SIGUSR1 to the node, which signals our entrypoint to boot
-// see images/node/entrypoint
-func (n *Node) SignalStart() error {
-	return docker.Kill("SIGUSR1", n.name)
```

Comment:
we should probably remove the docker.Kill call since this was the only user afaik. that can maybe be another PR though, this PR is already a bit big